analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -6,7 +6,7 @@ repository:
     revision: "d302b1e93963c81ed511e072a52e95251b5d078b"
```

Comment:
commit message: to me it is not clear which string comparison issues you are referring to

Reply:
The ones that are removed by this commit :wink: E.g. in [this original code](https://github.com/heremaps/oss-review-toolkit/pull/1616/files?file-filters%5B%5D=.kt#diff-4a7e540a7cebb1fb4e08363b36647e19L182) it was completely unclear whether you should compare `type` to "Git" or "git". This is the general issue that gets solved by this PR. But I'll clarify that in the commit message.